docs/topics/generic.rst

```diff
@@ -213,7 +213,7 @@ After these two steps you will use the ``reset_confirm_pin`` endpoint. It works
 the same way as the ``confirm_pin`` endpoint but instead checks against the
 buyer's ``new_pin`` rather than their ``pin``:
 
-.. htp:post::: /generic/reset_confirm_pin/
+.. http:post::: /generic/reset_confirm_pin/
```

Comment:
heh, oops. We should really start building docs as part of the test suite (to check for errors).